packages/flutter/lib/src/material/drawer.dart

```diff
@@ -220,6 +221,10 @@ class DrawerController extends StatefulWidget {
   /// {@endtemplate}
   final DragStartBehavior dragStartBehavior;
 
+  /// Set a color to use for the scrim that obscures primary content
```

Comment:
Please rephrase the documentation to better describe the behavior. Something like

```
The color to use for the scrim that obscures primary content while a drawer is open.

By default, the color is [Colors.black54]
```

Reply:
done